### ISSUE TITLE
Stuck after Authentication step when starting Lita

### ISSUE THREAD

**Issue by Brutalbeard** [ringcentral-ruby/lita-glip#3]
When running app through Terminal, application Authenticates without issue, and then sticks. Pubnub.log includes an error, copied below.

```
D, [2017-03-07T11:50:02.173931 #41648] DEBUG -- Pubnub: Fire before fire_callback
D, [2017-03-07T11:50:02.174152 #41648] DEBUG -- Pubnub::Event: Requested URI: http://ps.pndsn.com/v2/subscribe/sub-c-b8b9cd8c-e906-11e2-b383-02ee2ddab7fe/17257211649296_9616a522/0?pnsdk=PubNub-Ruby%2F4.0.17&t=%7B%22r%22%3A1%2C%22t%22%3A%2214889053246246831%22%7D&uuid=f585303b-5b37-46ae-b734-5cf6ce71f977
D, [2017-03-07T11:50:02.174583 #41648] DEBUG -- Pubnub::Event: Event#handle
D, [2017-03-07T11:50:02.174642 #41648] DEBUG -- Pubnub::SubscribeEvent::Formatter: Formatting non-errror envelopes
D, [2017-03-07T11:50:02.175193 #41648] DEBUG -- Pubnub: Firing callbacks for Pubnub::Subscribe
D, [2017-03-07T11:50:02.175227 #41648] DEBUG -- Pubnub::Subscriber: Firing callback from listeners
D, [2017-03-07T11:50:02.175251 #41648] DEBUG -- Pubnub::Subscriber: Firing callbacks from listener 'ringcentral'.
**E, [2017-03-07T11:50:03.157309 #41648] ERROR -- Pubnub::Subscriber: Error while calling callback #<NameError: undefined local variable or method `glip_user' for Lita::Adapters::Glip::UserCreator:Class
Did you mean?  glip_user_name
               glip_sdk>**
D, [2017-03-07T11:50:03.157502 #41648] DEBUG -- Pubnub: Fired event Pubnub::Subscribe
D, [2017-03-07T11:50:03.157537 #41648] DEBUG -- Pubnub: #send_request start
D, [2017-03-07T11:50:03.157554 #41648] DEBUG -- Pubnub: #request_dispatcher
D, [2017-03-07T11:50:03.157571 #41648] DEBUG -- Pubnub::Client: Looking for requester for async subscribe_event
D, [2017-03-07T11:50:03.157740 #41648] DEBUG -- Pubnub::Event: Requested URI: http://ps.pndsn.com/v2/subscribe/sub-c-b8b9cd8c-e906-11e2-b383-02ee2ddab7fe/17257211649296_9616a522/0?pnsdk=PubNub-Ruby%2F4.0.17&t=%7B%22r%22%3A1%2C%22t%22%3A%2214889054021302455%22%7D&uuid=f585303b-5b37-46ae-b734-5cf6ce71f977
```

**Comment by grokify** [comment]
Thanks for reporting this. Should be fixed in release `v0.0.3`:

* Github: https://github.com/ringcentral-ruby/lita-glip/releases/tag/v0.0.3
* RubyGems: https://rubygems.org/gems/lita-glip

Please try and let me know if it works for you now.

**Comment by Brutalbeard** [comment]
No longer hanging after authentication. App reads data from glip chat room, but responses only show in terminal window, and make it back to glip. i'll open a separate issue.
